spec/reset_password_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper'
+
+describe 'password reset', type: :feature, js: true do
+  it 'resets password at LOA1' do
+    email_address = create_new_account
+
+    visit idp_logout_url
+
+    visit idp_reset_password_url
+
+    expect(page).to have_content('Forgot')
```

Comment:
My assumptions:

* each test will take a relatively long time to run
* each test creates a new account on the server
* when an rspec step fails because content was not found, I personally find it helpful to know that I didn't reach the expected page *before* I attempt to act on the page I did expect.